sdk/include/ndk/umtypes.h

```diff
@@ -108,12 +108,12 @@ Header Name:
 //
 // Limits
 //
-#define MINCHAR                         (-128)
-#define MAXCHAR                         127
-#define MINSHORT                        (-32768)
-#define MAXSHORT                        32767
-#define MINLONG                         (-2147483648)
-#define MAXLONG                         2147483647
+#define MINCHAR                         0x80
+#define MAXCHAR                         0x7f
+#define MINSHORT                        0x8000
+#define MAXSHORT                        0x7fff
+#define MINLONG                         0x80000000
+#define MAXLONG                         0x7fffffff
```

Comment:
With `L` suffix maybe ? At least for the 32bits values.

Reply:
There is no reason for it, because on Windows both long and int are 32 bit, so there is no difference (unless you have a stupid compiler like GCC and want to mix a ponter to long with a pointer to int, which technically is the exact same thing, but GCC doesnt like it, but that is not an issue here). And Windows doesn't have it either.